components/table/Table.tsx

```diff
@@ -124,22 +184,83 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
     childrenColumnName: 'children',
   };
 
-  CheckboxPropsCache: {
-    [key: string]: any;
-  };
+  static getDerivedStateFromProps(nextProps: TableProps<any>, prevState: TableState<any>) {
```

Comment:
You need `react-lifecycles-compat` to polyfill Table for react 15.

Reply:
Yes, I figured, but the tests are failing with that in place as well. 😞